sechub-api-java/src/test/java/com/mercedesbenz/sechub/api/DefaultSechubClientWireMockTest.java

```diff
@@ -59,8 +89,8 @@ public void fetch_sechub_status_with_basic_auth() throws Exception {
         /* test */
         verify(getRequestedFor(urlEqualTo("/api/admin/status")));
         assertNotNull(status);
-        assertEquals("true", status.getStatusInformationMap().get("status.scheduler.enabled"));
-        assertEquals("2", status.getStatusInformationMap().get("status.scheduler.jobs.all"));
+        assertTrue(status.scheduler().isEnabled());
+        assertEquals(2, status.jobs().all());
```

Comment:
You added additional entries inside the test json file - so please test them as well.